cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/snowbridge.rs

```diff
@@ -535,3 +537,52 @@ fn send_token_from_ethereum_to_asset_hub_fail_for_insufficient_fund() {
 		assert_err!(send_inbound_message(make_register_token_message()), Arithmetic(Underflow));
 	});
 }
+
+#[test]
+fn transact_from_ethereum_to_penpal() {
```

Comment:
We also need two negative tests:
* Transact failing because not enough fees passed
* Transact failing because `weightAtMost` not large enough

Reply:
https://github.com/Snowfork/polkadot-sdk/pull/114/commits/626f2cb26782c71264d3f32ff0b0cb1695fab6ca